### ISSUE TITLE
Glide with dragon

### ISSUE THREAD

**Issue by PlatinumFox** [AlexModGuy/Ice-and-Fire-Suggestions#19]
Ok, so I was thinking about HTTYD and scenes from the second movie with the glide suit and that got me to thinking. Minecraft 1.10.2 has elytra in it. And the dragon mounts mod is open source and claims that when flying with elytra the dragon will glide next to you (I have not tested this myself). So I thought what if the developers somehow implemented a feature like that with the dragons where you can glide with wings and they will follow and glide along with you in mid air. That leads me on to my next idea. Wearing vanilla elytra is very very ugly unless the only thing you are wearing is the pair of elytra. So what if there was a new item/armor piece that could be crafted that was a dragon armor chest plate in the middle of the crafting grid, a Pair of elytra under it, and then maybe some slime balls or something filling up the other 7 slots. (Let me try to demonstrate below)

H = Chest Plate
V = Elytra
O = SlimeBall

O O O
O H O
O V O

So this would basically craft a Dragon Wing Suit where if you where falling it and pressed spacebar it would activate like elytra and if you where wearing full dragon armor you would look amazing while doing it all the while having your dragon gliding next to you at your side.

**Comment by PlatinumFox** [comment]
This was something that i posted to the forums a long time ago thought i would repost it as it would also go good with #8

**Comment by LordDecay26** [comment]
yup... this also needs to be a thing, it might not seem necessary but i feel that the developers of this mod are definitely missing out on an opportunity if they do not do this!
